docs/backend/guides/how-to/upgrade-node.md

```diff
@@ -9,7 +9,7 @@ sidebar_position: 7
 3. In the transcom/mymove repo, create a PR to update node. [Here's an example](https://github.com/transcom/mymove/pull/6878) && [this PR](https://github.com/transcom/mymove/pull/6904)
 
 Things to modify:
-* Bump the `.node_version`
+* Bump the nodejs version in `.tool-versions` and `.node_version` (some folks may still be using nodenv to manage node)
```

Comment:
@rogeruiz I think that makes the most sense. Deleting it would be a forcing function for engineers to udpate, which normally I'd advocate for, especially if it means a tidier codebase. In this case, I think approaching go-live is probably the wrong time to throw wrenches at engineers. 

So this allows folks to use _either_ tool for now, but folks should definitely stick to one or the other.

When someone sets up for the first time, however, they'll be setup with asdf rather than nodenv